cmake/legacy.cmake

```diff
@@ -211,29 +211,26 @@ ExternalProject_Add(flatbuffers
 
 if (NOT PACKAGE_SET STREQUAL fairmqdev)
   list(APPEND packages fairmq)
-  set(fairmq_version "1.4.44")
+  set(fairmq_version "1.4.50")
   ExternalProject_Add(fairmq
     GIT_REPOSITORY https://github.com/FairRootGroup/FairMQ GIT_TAG v${fairmq_version}
-    ${CMAKE_DEFAULT_ARGS} CMAKE_ARGS
-      "-DBUILD_DDS_PLUGIN=OFF"
-      "-DBUILD_SDK_COMMANDS=OFF"
-      "-DBUILD_SDK=OFF"
-    DEPENDS asio boost fairlogger flatbuffers zeromq ${extract_source_cache_target}
+    ${CMAKE_DEFAULT_ARGS}
+    DEPENDS asio boost fairlogger zeromq ${extract_source_cache_target}
     ${LOG_TO_FILE}
   )
 
-  list(APPEND packages odc)
-  set(odc_version "0.62")
-  ExternalProject_Add(odc
-    GIT_REPOSITORY https://github.com/FairRootGroup/ODC GIT_TAG ${odc_version}
-    ${CMAKE_DEFAULT_ARGS} CMAKE_ARGS
-      "-DBUILD_GRPC_CLIENT=OFF"
-      "-DBUILD_GRPC_SERVER=OFF"
-      "-DBUILD_EPN_PLUGIN=OFF"
-      "-DBUILD_EXAMPLES=OFF"
-    DEPENDS boost dds fairlogger fairmq ${extract_source_cache_target}
-    ${LOG_TO_FILE}
-  )
+  # list(APPEND packages odc)
+  # set(odc_version "0.62")
+  # ExternalProject_Add(odc
+    # GIT_REPOSITORY https://github.com/FairRootGroup/ODC GIT_TAG ${odc_version}
+    # ${CMAKE_DEFAULT_ARGS} CMAKE_ARGS
+      # "-DBUILD_GRPC_CLIENT=OFF"
+      # "-DBUILD_GRPC_SERVER=OFF"
+      # "-DBUILD_EPN_PLUGIN=OFF"
+      # "-DBUILD_EXAMPLES=OFF"
+    # DEPENDS boost dds fairlogger fairmq ${extract_source_cache_target}
+    # ${LOG_TO_FILE}
+  # )
```

Comment:
What about moving the odc-setup into some `add_package_odc` macro and just not call it?

Reply:
ya, I should probably just have deleted the section. It's just noise. Will do on next PR.